src/protocol/definitions/common.json

```diff
@@ -132,6 +132,37 @@
                         "description": "The protocol version"
                     }
                 ]
+            },
+            "device_information": {
+                "id": "4",
```

Comment:
You should add this before id 5, this breaks generate code and documentation order

Reply:
done